data-reporting-service/organization-service/src/test/java/gov/cdc/etldatapipeline/organization/OrganizationDataProcessTests.java

```diff
@@ -0,0 +1,103 @@
+    OrganizationSp orgSp;
+
+    @BeforeEach
+    public void setup() throws JsonProcessingException {
+        orgSp = objectMapper.readValue(readFileData("orgcdc/orgSp.json"), OrganizationSp.class);
+    }
+
+    @Test
+    public void OrganizationNameProcessTest() {
+        Name[] name = utilHelper.deserializePayload(orgSp.getOrganizationName(), Name[].class);
+        Name expected = Name.builder()
+                .onOrgUid(10036000L)
+                .organizationName("Autauga County Health Department")
+                .build();
+
+        assertEquals(expected.toString(), name[0].toString());
+    }
+
+    @Test
+    public void OrganizationAddressProcessTest() {
+        Address[] addr = utilHelper.deserializePayload(orgSp.getOrganizationAddress(), Address[].class);
+        Address expected = Address.builder()
+                .addrElpCd("O")
+                .addrElpUseCd("WP")
+                .addrPlUid(10036001L)
+                .streetAddr1("219 North Court Street")
+                .streetAddr2("Unit#1")
+                .city("Prattville")
+                .zip("36067-0000")
+                .cntyCd("01001")
+                .state("01")
+                .cntryCd("840")
+                .state_desc("Alabama")
+                .county("Autauga County")
+                .addressComments("Testing address Comments!")
+                .build();
+
+        assertEquals(expected.toString(), addr[0].toString());
+    }
+
+    @Test
+    public void OrganizationPhoneProcessTest() {
+        Phone[] phn = utilHelper.deserializePayload(orgSp.getOrganizationTelephone(), Phone[].class);
+        Phone expected = Phone.builder()
+                .phTlUid(10615102L)
+                .phElpCd("PH")
+                .phElpUseCd("WP")
+                .telephoneNbr("3343613743")
+                .extensionTxt("1234")
+                .emailAddress("john.doe@test.com")
+                .phone_comments("Testing phone Comments!")
+                .build();
+
+        assertEquals(2, phn.length);
+        assertEquals(expected.toString(), phn[1].toString());
+    }
+
+    @Test
+    public void OrganizationEntityProcessTest() {
+        Entity[] ets = utilHelper.deserializePayload(orgSp.getOrganizationEntityId(), Entity[].class);
+        Entity expected = Entity.builder()
+                .entityUid(10036000L)
+                .typeCd("FI")
+                .recordStatusCd("ACTIVE")
+                .rootExtensionTxt("A4646")
+                .entityIdSeq("1")
+                .assigningAuthorityCd("OTH")
+                .build();
+
+        assertEquals(2, ets.length);
+        assertEquals(expected.toString(), ets[0].toString());
+    }
+
+    @Test
+    public void OrganizationFaxProcessTest() {
+        Fax[] fax = utilHelper.deserializePayload(orgSp.getOrganizationFax(), Fax[].class);
+        Fax expected = Fax.builder()
+                .faxTlUid(1002L)
+                .faxElpCd("fax-cd-1002")
```

Comment:
Are these mock codes for faxElpCd and faxElpUseCd?

Reply:
> Are these mock codes for faxElpCd and faxElpUseCd?

It's `faxElpCd` and  `faxTlUid`, got them from `sp_organization_event` output, `organizationFax` nested field:

```"organizationFax": "[{"fax_tl_uid":1002,"fax_elp_cd":"fax-cd-1002",...}]```